pubsub/cloud-client/subscriber.py

```diff
@@ -161,16 +161,16 @@ def update_subscription(project_id, subscription_name, endpoint):
     # [END pubsub_update_push_configuration]
 
 
-def receive_messages(project_id, subscription_name):
+def receive_messages(project_id, subscription_name, timeout=None):
     """Receives messages from a pull subscription."""
     # [START pubsub_subscriber_async_pull]
     # [START pubsub_quickstart_subscriber]
-    import time
-
     from google.cloud import pubsub_v1
 
     # TODO project_id = "Your Google Cloud Project ID"
     # TODO subscription_name = "Your Pub/Sub subscription name"
+    # TODO timeout = "How long the subscriber should listen for messages in
```

Comment:
The idea of these TODOs is to give users a concrete example of how these variables should look like if they uncomment them. I personally omit the `TODO` word, but that's more on a consistency with the product. I guess the timeout is a number, so it should be something like:

```
# TODO timeout = 5.0  # How long the subscriber should listen for messages (seconds)
```

Reply:
Done.